test/unit/bindings.test.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
```

Comment:
I moved the root tests to test/unit to avoid all the root test/ eslint craziness.